### PR TITLE
feat: mobile responsive

### PR DESCRIPTION
**Pull request by mansipadgelwar**
Make NoTress responsive

### CODE REVIEW

File: src/components/SideBar/SideBar.css [hunk 1]
```diff
@@ -81,3 +81,59 @@ a:-webkit-any-link {
 .App.dark .notes-side-links {
   color: var(--white);
 }
+
+/* .desktop-hide {
```
Comment (t007rushi) [inline src/components/SideBar/SideBar.css]: comment to remove

File: src/components/SideBar/SideBar.css [hunk 2]
```diff
@@ -81,3 +81,59 @@ a:-webkit-any-link {
...
+
+  .notes-sidebar-list {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    gap: 0;
+  }
+
+  .profile-details {
+    display: none !important;
```
Comment (t007rushi) [inline src/components/SideBar/SideBar.css:118]: avoid !important

**Review by t007rushi — APPROVED** [review]
Good naming camelCase everywhere seems readable
good to merge

File: src/components/SideBar/SideBar.css [hunk 3]
```diff
@@ -81,3 +81,59 @@ a:-webkit-any-link {
...
+  display: none;
+}
+
+@media screen and (max-width: 567px) {
+  .notes-side-bar {
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    top: auto;
+    max-width: 767px;
```
Comment (simran61) [inline src/components/SideBar/SideBar.css]: can use relative units instead of pixels

**Review by simran61 — APPROVED** [review]
Looks good

**Pull request merged**